src/test/java/nextstep/subway/acceptance/path/steps/PathAcceptanceSteps.java

```diff
@@ -15,12 +16,18 @@ public class PathAcceptanceSteps {
   private PathAcceptanceSteps() {}
 
   public static ExtractableResponse<Response> 경로_조회_요청(Station source, Station target) {
-    PathRequest request = new PathRequest(source.getId(), target.getId());
+    PathRequest request = PathRequest.of(source.getId(), target.getId(), PathType.DISTANCE);
     return RestAssured.given()
         .log()
         .all()
         .accept(MediaType.APPLICATION_JSON_VALUE)
-        .queryParams("source", request.getSource(), "target", request.getTarget())
+        .queryParams(
+            "source",
+            request.getSource(),
+            "target",
+            request.getTarget(),
+            "type",
+            request.getType().name())
```

Comment:
```suggestion
        .queryParams(
            "source", request.getSource(),
            "target", request.getTarget(),
            "type", request.getType().name())
```

이런형태인게 가독성이 더 좋을 것 같아요 😢